nixos/modules/services/misc/mediatomb.nix

```diff
@@ -206,20 +219,20 @@ in {
 
       dataDir = mkOption {
         type = types.path;
-        default = "/var/lib/mediatomb";
+        default = "/var/lib/" + name;
```

Comment:
This will cause existing installations to mysteriously break. Either leaving the default to `mediatomb` or explicitly mentioning this in release notes is a good idea.

Reply:
Yes, well this was the intention since it has to be done sooner or later. I'll add it to the release notes to ease the transition.